test/src/e2e_vm_tests/test_programs/should_pass/language/method_on_primitives/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
```

Comment:
This `.gitignore` file is not needed and should be deleted. (We need a lint here...)

Reply:
Done.